### DISCUSSION TITLE
No previews in 0.3.1

### DISCUSSION THREAD

**Discussion by doorbuster** [sxyazi/yazi#1515] [1. Q&A] [answered]
I've been enjoying yazi for a couple of months.  Since updating to 0.3.1 yesterday, there are no previews of any kind, and I can't open files.  I'm on EndeavourOS and Kitty is my terminal.  Any ideas?
The yamb bookmark plugin isn't working either...

**Comment by AnirudhG07** [comment]
Can you share your `yazi --debug`? maybe your dotfiles for yazi would be useful as well(if you are comfortable sharing)

**Comment by doorbuster** [comment]
Here they are.  I tried hiding them from yazi, as I thought Kitty handled some previewing out of the box.  No luck.
[yazidotfiles.zip](https://github.com/user-attachments/files/16651922/yazidotfiles.zip)

**Comment by doorbuster** [comment]
This might be a clue.   I used lf before yazi and still have it installed.  The preview box in lf shows this error (all that fits in the preview box).  I didn't think yazi used ctpv though.

ctpv: src/ctpv.c:50: internal error: 'magic_load(magic, NULL)  !=0':

**Comment by sxyazi** [comment]
I couldn't reproduce the issue using your configuration file. Have you tried starting Yazi with just the default config (`mv ~/.config/yazi ~/.config/yazi.bak`) to see if it's a bug in Yazi itself or something caused by the config? If it’s the config, you could try adding things back bit by bit to narrow down the problem.

**Reply by doorbuster** [reply]
I did that to no avail, so it's apparently outside of Yazi.  I get the same results in Konsole, if it works similarly.  Besides the plugins I installed, are there other dependencies, or does Kitty handle the previews internally?

**Reply by doorbuster** [reply, marked as answer]
I was able to get the full error message from ctpv, and it said that no magic files could be found.  Then I remembered struggling with imhex, which also uses magic files.  And I remembered that I had found 'magic.mgc' in a folder I didn't expect it to be.  I didn't realize it wasn't only for imhex.  Thankfully I renamed it rather than delete it.  When I fixed that filename, if fixed Yazi and lf, just like magic!  I don't know what a magic file is, but I've learned not to mess with it!  Thanks for your help.
